docs/src/recipes/appendix.asciidoc

```diff
@@ -142,7 +142,7 @@ The following example assumes that the edges point in the `OUT` direction. Assum
 ----
 g.V().where(without("x")).as("a").
   outE().as("e").inV().as("b").
-  filter(bothE().where(neq("e")).otherV().where(eq("a"))).aggregate(local, "x").
+  filter(bothE().where(neq("e")).otherV().where(eq("a"))).local(aggregate("x")).
```

Comment:
i think i've asked this elsewhere, but again, have we tested these examples manually to be sure our results are the same just by adding the `local` wrapping?